styles.css

```diff
@@ -454,6 +454,207 @@ input[type=name] {
   align-items: center;
 }
 
+#work-section {
+  font-family: sans-serif;
+  font-weight: 500;
+  margin-top: 70px;
+}
+
+#box-1 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  margin-top: 40px;
+}
+
+#head {
+  border-bottom: 1px solid #646369;
+}
+
+#head h2 {
+  width: 50%;
+  font-size: 40px;
+  line-height: 54px;
+  color: #7d73fc;
+  margin-bottom: 30px;
+}
+
+#box-1 img {
+  width: 100%;
+  margin-bottom: 40px;
+  cursor: pointer;
+}
+
+#box-1 img:hover {
+  transform: scale(0.99);
+  transform: rotate(2deg);
+}
+
+#box-1 h1 {
+  font-family: sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-bottom: 40px;
+}
+
+#box-1 p {
+  font-family: sans-serif;
+  font-weight: 400;
+  color: #dcdce8;
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 40px;
+}
+
+#box-1 ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  margin-bottom: 40px;
+}
+
+#box-2 ul li:nth-child(2),
+#box-2 ul li:nth-child(1),
+#box-2 ul li:nth-child(3) {
+  border-right: 1px solid #646369;
+  padding: 10px 20px;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+#box-2 ul li:nth-child(4) {
+  padding: 10px 20px;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+#button {
+  width: 20%;
+  font-family: sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 20px;
+  color: #fff;
+  background: #6a61d7;
+  border-radius: 1px;
+  padding: 12px 12px;
+  border: 1px solid #6a61d7;
+  transition: 0.2s;
+  cursor: pointer;
+}

```

Comment:
- [ ] Please know that the buttons are small and don't match the design, this is how they look:
![image](https://user-images.githubusercontent.com/20567503/185741169-8248ff54-82d7-41fb-8205-5da9d04cbe7d.png)
They should look like this:
![image](https://user-images.githubusercontent.com/20567503/185741175-09b45f03-7201-4ced-9810-c4d0011c0191.png)

Kindly fix this as well. 😁

- [ ] Please add the missing styling for the different states of the buttons provided by the design:
![image](https://user-images.githubusercontent.com/20567503/185741688-9bddd031-d01a-4449-aff1-e9fac4c9ea66.png)